packages/amplify-graphql-conversation-transformer/src/__tests__/amplify-graphql-conversation-transformer.test.ts

```diff
@@ -324,7 +324,7 @@ function transform(
     hasManyTransformer,
     hasOneTransformer,
     belongsToTransformer,
-    new ConversationTransformer(modelTransformer, hasManyTransformer, belongsToTransformer, authTransformer, functionMap),
+    new ConversationTransformer(modelTransformer, hasManyTransformer, belongsToTransformer, authTransformer, undefined, functionMap),
```

Comment:
Doesn't have to happen on this PR, but it's probably time to create a `ConversationTransformerOptions` type to support named inputs to the constructor -- the argument list is pretty unwieldy